content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,78 @@
 Assign leads based on scoring
 =============================
 
-With *Leads Scoring* you can automatically rank your leads based on
-selected criterias.
+With *Lead Scoring* you can automatically rank your leads based on
+selected criteria.
+
+For example, you could score customers higher when they come from the same country higher as your
+company or from visiting specific pages on your website.
 
-For example you could score customers from your country higher or the
-ones that visited specific pages on your website.
 
 Configuration
 =============
 
-To use scoring, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+To use scoring, install the free module *Lead Scoring* under your *Apps* page (only available in
+Odoo Enterprise).
 
-.. image:: media/lead_scoring01.png
+.. image:: lead_scoring/lead-scoring-module.png
    :align: center
+   :alt: Lead Scoring Module Installation
 
 Create scoring rules
 ====================
 
-You now have a new tab in your *CRM* app called *Leads Management*
-where you can manage your scoring rules.
+You now have a new tab in your *CRM* app called *Leads* where you can manage your scoring rules.
 
-Here's an example for a Canadian lead, you can modify for whatever
-criteria you wish to score your leads on. You can add as many criterias
-as you wish.
+Here's an example of a Canadian lead. You can modify your rules for whatever criteria you want to
+score leads on. You can add as many criteria as you would like.
 
-.. image:: media/lead_scoring02.png
+.. image:: lead_scoring/scoring-example.png
    :align: center
+   :alt: Score Ruling Example
 
-Every hour every lead without a score will be automatically scanned and
-assigned their right score according to your scoring rules.
+Every lead without a score is automatically be scanned every hour and is assigned the correct
+score according to your configured scoring rules.
 
-.. image:: media/lead_scoring03.png
+.. image:: lead_scoring/scoring-section-on-lead.png
    :align: center
+   :alt: Scoring Section on a Lead
 
 Assign leads
 ============
 
-Once the scores computed, leads can be assigned to specific teams using
-the same domain mechanism. To do so go to :menuselection:`CRM --> Leads Management --> Team Assignation`
-and apply a specific domain on each team. This domain can include scores.
+Once the score is computed, leads can be assigned to specific teams using the same domain
+mechanism. To do so, head to :menuselection:`CRM --> Leads --> Team Assignment` and apply a
```

Comment:
It feels weird to mention the "same domain mechanism" because you didn't mention "domains" earlier.